pkg/daemon/ceph/client/image.go

```diff
@@ -33,9 +33,10 @@ const (
 )
 
 type CephBlockImage struct {
-	Name   string `json:"image"`
-	Size   uint64 `json:"size"`
-	Format int    `json:"format"`
+	Name     string `json:"image"`
+	Size     uint64 `json:"size"`
+	Format   int    `json:"format"`
+	InfoName string `json:"name"`
```

Comment:
Had to add the `InfoName` field because in `rbd info --format json` output the image name is in the `name` field (see tests; as opposed to `rbd ls` output, where the image name is in the `image` field).